h/emails/reply_notification.py

```diff
@@ -20,11 +21,8 @@ def generate(request, notification):
     if not document_title:
         document_title = notification.parent.target_uri
 
-    parent_url = request.route_url('annotation', id=notification.parent.id)
-    reply_url = request.route_url('annotation', id=notification.reply.id)
-
-    reply_user_url = request.route_url('stream.user_query',
-                                       user=notification.reply_user.username)
+    reply_url = (links.incontext_link(request, notification.reply) or
```

Comment:
Done.

Why do you say this is clearer though? Is it because the `or` version requires wrapping in parens because it doesn't fit on one line?

In general I would have thought that `foo = thing_1 or thing_2` is quite Pythonic.